dts/arm/nxp/nxp_rt.dtsi

```diff
@@ -800,19 +800,27 @@
 			#pinmux-cells = <2>;
 			reg = <0x40384000 0x4000>;
 			clocks = <&ccm IMX_CCM_SAI1_CLK 0x7C 2>;
-			pre-div = <0>;
-			podf = <63>;
-			pll-clocks = <&anatop 0x70 0xC000 0>,
-					<&anatop 0x70 0x7F 32>,
-					<&anatop 0x70 0x18 1>,
-					<&anatop 0x80 0x3FFFFFFF 77>,
-					<&anatop 0x90 0x3FFFFFFF 100>;
+			// Audio PLL Output Frequency is determined by:
+			// (Fref * (DIV_SELECT + NUM/DENOM)) / POST_DIV
+			// = (24MHz * (32 + 77 / 100)) / 1 = 786.48 MHz
+		       pll-clocks = <&anatop 0x70 0xC000 0>,
+				      <&anatop 0x70 0x7F 32>,
+				      <&anatop 0x70 0x180000 1>,
+				      <&anatop 0x80 0x3FFFFFFF 77>,
+				      <&anatop 0x90 0x3FFFFFFF 100>;
 			pll-clock-names = "src", "lp", "pd", "num", "den";
+			// The maximum input frequency into the SAI mclk input is 300MHz
+			// Based on this requirement, pre-div must be at least 3
```

Comment:
@NickolasLapp can you replace all C99 style `// ` comments in this PR with `/* */` C89 style comments? CI isn't catching it but generally only C89 style comments are allowed upstream

Reply:
Comments are replaced! Thanks for catching